.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+ [ ] I have searched the [PR](https://github.com/Marketing-Automation-Systems/jellyfish/pulls/) of this repository and believe that this is not a duplicate.
+
+<!-- Checked checkbox should look like this: [x] -->
+
+**Describe the PR**
+
+This PR closes [#ISSUE_NUMBER](ISSUE LINK)
```

Comment:
You don't need to use [#ISSUE_NUMBER](ISSUE LINK), just #ISSUE_number and github will autocomplete for you. For instance, `Closes #13`

Reply:
Yes I know, I tried to make more explicit, but I'll change.